src/index.html

```diff
@@ -6,6 +6,9 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Furnitures shop</title>
     <link rel="stylesheet" href="./sass/style.scss" />
+
+    <script src="//code.jquery.com/jquery-3.2.1.min.js"></script>
```

Comment:
Dlaczego dodajemy to w `head`? Powinno się to znajdować na końcu przed zamknięciem znacznika `body`

Reply:
Niestety pojawił się problem z wgraną biblioteką Bootstrap, że nie rozpoznawała jQuery. Michał to dzisiaj odkrył, bo nie tylko ja zgłaszałem ten problem. Nie rozumiałem gdzie tkwi problem, więc podczepiałem inne biblioteki trochę na ślepo.